internal/services/cockpit/helpers_cockpit.go

```diff
@@ -21,14 +24,34 @@ const (
 )
 
 // NewAPI returns a new cockpit API.
-func NewAPI(m interface{}) (*cockpit.API, error) {
-	api := cockpit.NewAPI(meta.ExtractScwClient(m))
+func NewAPI(m interface{}) (*cockpitv1beta1.API, error) {
```

Comment:
Could we manage only the v1 api? Is there a premium to maintain both APIs?

Reply:
As we want to keep the legacy cockpit resource for some time, we have to leave the v1beta1 api to make the calls, in v1 there's no longer any notion of cockpit